distributed/bokeh/tests/test_scheduler_bokeh_html.py

```diff
@@ -68,6 +68,10 @@ def test_prefix(c, s, a, b):
 def test_prometheus(c, s, a, b):
     pytest.importorskip('prometheus_client')
     http_client = AsyncHTTPClient()
-    response = yield http_client.fetch('http://localhost:%d/metrics'
-                                       % s.services['bokeh'].port)
-    assert response.code == 200
+
+    # request data twice since there once was a case where metrics got registered multiple times resulting in
+    # prometheus_client errors
+    for _ in range(2):
+        response = yield http_client.fetch('http://localhost:%d/metrics'
+                                           % s.services['bokeh'].port)
+        assert response.code == 200
```

Comment:
As a smoke test, I would recommend to also check for the existence of at least one metric (e.g. anything starting with `dask_distributed`).